honestcash-v2/src/app/editor/editor.module.ts

```diff
@@ -62,7 +62,6 @@ const routes: Routes = [
   imports: [
     FormsModule,
     RouterModule.forChild(routes),
-    NgbModule,
```

Comment:
NgbModule is now shared via sharedModule (because even if I add a tooltip, I have to include it so I'll later look into possibility of importing NgbModule individually